### ISSUE TITLE
Hyperparameters about ResNetSE34V2

### ISSUE THREAD

**Issue by whitegon** [clovaai/voxceleb_trainer#76]
Hello,
I want to reproduce the result of ResNetSE34V2,
Can I get the hyperparameters about this model?
like, 
max_frames 200
eval_frames 300
batch_size 64
max_seg_per_spk 100
nDataLoaderThread 5
augment True
test_interval 10
max_epoch 500
trainfunc softmaxproto
optimizer adam
scheduler steplr
lr 0.001
lr_decay 0.95
weight_decay 0
hard_prob 0.5
hard_rank 10
margin 1
scale 15
nPerSpeaker 2
n_mels 64
log_input True
model ResNetSE34V2
encoder_type ASP
nOut 512
eval False

Thank you!

**Comment by whitegon** [comment]
See #71
